src/View/Exit/show.html.twig

```diff
@@ -0,0 +1,84 @@
+{% extends 'layout.html.twig' %}
+
+{% block content %}
+
+	{% set exitId = "" %}
+	{% set exitName = "" %}
+	{% set exitDepartment = "" %}
+	{% set exitCountry = "" %}
+	{% set exitHeight = "" %}
+	{% set exitAccessDuration = "" %}
+	{% set exitGpsCoordinates = "" %}
+	{% set exitAcces = "" %}
+	{% set exitRemark = "" %}
+	{% set exitImage = "" %}
+	{% set exitVideo = "" %}
+	{% set typeJumpName = "" %}
+
+	{% for item in exit %}
+
+		{% set exitId = item.id %}
+		{% set exitName = item.name %}
+		{% set exitDepartment = item.department %}
+		{% set exitCountry = item.country %}
+		{% set exitHeight = item.height %}
+		{% set exitAccessDuration = item.access_duration %}
+		{% set exitGpsCoordinates = item.gps_coordinates %}
+		{% set exitAcces = item.acces %}
+		{% set exitRemark = item.remark %}
+		{% set exitImage = item.image %}
+		{% set exitVideo = item.video %}
+		{% set typeJumpName = typeJumpName ~ ' ' ~ item.type_jump_name ~ ' ! ' %}
```

Comment:
No ! I get why you do that for typeJumpName cause you have multiple jump on an exit. BUT for the rest of the fields you shouldn't use all this variables.

The best solution is to change the function selectExitTypeJump in your manager to only send you jumpNames by exit id. Then in your controller you'll send to twig, 1 exit (get with simple selectById) and an array of jump. It will mor clear to use in the template